src/DynamoCore/Library/LibraryCustomization.cs

```diff
@@ -98,8 +99,12 @@ public static LibraryCustomization GetForAssembly(string assemblyPath, IPathMana
                     return false;
 
                 resourceAssemblyPath = fn + Configurations.IconResourcesDLL;
-                // Side-by-side customization dll not found, try other resolution paths.
-                return pathManager.ResolveLibraryPath(ref resourceAssemblyPath);
+
+                if (useAdditionalPaths)
+                    // Side-by-side customization dll not found, try other resolution paths.
+                    return pathManager.ResolveLibraryPath(ref resourceAssemblyPath);
+                else
+                    return false;
```

Comment:
This looks good, but if I can have additional request, that would be this (sorry for being nit-picking!):

```csharp
if (!useAdditionalPaths) // Should only look up alongside the main assembly.
    return false;
    
// Side-by-side customization dll not found, try other resolution paths.
return pathManager.ResolveLibraryPath(ref resourceAssemblyPath);
```